### PR TITLE
fix(oauth2): validation on ttl

### PR DESCRIPTION
**Pull request by StarlightIbuki**
fix FTI-3250

[FTI-3250]: https://konghq.atlassian.net/browse/FTI-3250?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by chronolaw — APPROVED** [review]

File: kong/db/schema/typedefs.lua [hunk 1]
```diff
@@ -822,4 +823,9 @@ setmetatable(typedefs, {
 })
 
 
+typedefs.ttl = Schema.define {
+  type = "number",
+  between = { 0, DAO_MAX_TTL - 1 },
```
Comment (StarlightIbuki) [inline kong/db/schema/typedefs.lua:828]: Should be inclusive.

```suggestion
  between = { 0, DAO_MAX_TTL },
```

**Review by flrgh — COMMENTED** [review]
Code looks good--just needs a changelog entry for the bugfix.

File: CHANGELOG.md [hunk 2]
```diff
@@ -112,6 +112,8 @@
   [#9877](https://github.com/Kong/kong/pull/9877)
 - **JWT**: Deny requests that have different tokens in the jwt token search locations. Thanks Jackson 'Che-Chun' Kuo from Latacora for reporting this issue.
   [#9946](https://github.com/Kong/kong/pull/9946)
+- **OAuth2**: `refresh_token_ttl` is now limited between 0 and 100000000 by scheme. Previously the larger number causes requests to fail.
```
Comment (dndx) [inline CHANGELOG.md:115]:
```suggestion
- **OAuth2**: `refresh_token_ttl` is now limited between `0` and `100000000` by schema validator. Previously numbers that are too large causes requests to fail.
```

**Review by dndx — APPROVED** [review]

**Comment by dndx** [comment]
@StarlightIbuki "This branch cannot be rebased due to conflicts"

**Comment by StarlightIbuki** [comment]
> @StarlightIbuki "This branch cannot be rebased due to conflicts"

Conflict resolved.

File: CHANGELOG.md [hunk 3]
```diff
@@ -116,15 +116,15 @@
...
+- **OAuth2**: `refresh_token_ttl` is now limited between `0` and `100000000` by schema validator. Previously numbers that are too large causes requests to fail.
+  [#10068](https://github.com/Kong/kong/pull/10068)
 
 ### Changed
 
 #### Hybrid Mode
 
 - Revert the removal of WebSocket protocol support for configuration sync,
   and disable the wRPC protocol.
-  [#9921](https://github.com/Kong/kong/pull/9921)
```
Comment (chronolaw) [inline CHANGELOG.md:126]: Why did we remove this PR link? any explain?

Comment (StarlightIbuki) [reply CHANGELOG.md:126]: I don't know. Maybe a mistake when solving conflict. Let's add it back.

**Pull request merged**